torch/csrc/jit/interpreter.cpp

```diff
@@ -678,7 +700,13 @@ struct InterpreterStateImpl {
         }
     }
     current_pc = pc;
+    if (stack.size() > 0) {
```

Comment:
The stack is not guaranteed to be empty on entry in the interpreter. You can figure out how many things are returned by the interpreter by looking at the number of outputs in the original graph, and saving that number in Code. Also, there may be more than 1 output. 

```
if(num_outputs == 1)
  future->markCompleted(stack.back());
else
  future->markCompleted(Tuple::create(peekSlice(stack,0, num_outputs))
```